### PR TITLE
update on ticket "Implement the frontend page for "ok" using ppa and api faking#27"

### PR DESCRIPTION
**Pull request by Motouom**

### CODE REVIEW

**Comment by Motouom** [comment]
Hello @stephane-segning, @Arielpetit, @Christiantyemele   review PR

**Comment by Christiantyemele** [comment]
Good job! 👍  @Motouom but please undo the changes in the main.tsx

File: power-pay-frontend/src/main.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-import App from './App.tsx'
-import './index.css'
+import './App.css'  
+import OKPage from './components/okpage.tsx'
+
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
-    <App />
+    <OKPage />
```
Comment (Christiantyemele) [inline power-pay-frontend/src/main.tsx:10]: please undo this change its not neccessary in the main

**Review by Arielpetit — APPROVED** [review]
Good

**Review by Christiantyemele — APPROVED** [review]

File: power-pay-frontend/dev-dist/registerSW.js
Comment (stephane-segning) [inline power-pay-frontend/dev-dist/registerSW.js:1]: Why am I seeing this file in this PR?

File: power-pay-frontend/src/components/okpage.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/components/okpage.tsx:1]: The OK Page is supposed to be a layout. So you can split this into multiple components

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Review by stephane-segning — APPROVED** [review]

**Pull request closed without merging**